src/fontra/client/core/glyph-svg.js

```diff
@@ -51,3 +51,19 @@ function formatCoordinate(x, y, numerator, denominator, dx, dy) {
   y = Math.round(y * numerator + dy) / denominator;
   return `${x},${y}`;
 }
+
+export function pathToSVG(instance, bounds) {
+  const path = new SVGPath2D();
+  instance.path.drawToPath2d(path);
+  const pathString = path.getPath();
+
+  const width = bounds["xMax"] - bounds["xMin"];
+  const height = bounds["yMax"] - bounds["yMin"];
+
+  const svgString = `<svg xmlns="http://www.w3.org/2000/svg"
+                      width="${width}"
+                      height="${height}"
+                      viewBox="0 0 ${width} ${height}">
```

Comment:
I think the four fields need to be literally `xMin yMin xMax yMax`. The min values should be passed through `Math.floor()` and the max values through `Math.ceil()`.

Reply:
Perhaps the y values need work after the y-axis flip transformation. To test: save the svg as a file and open in Safari, it nicely shows how the svg is clipped.

Reply:
To clarify, "0 0" for the first two fields cause any part of the shape that has negative x coordinates to be cropped.